web/src/features/time/TimeHeader.tsx

```diff
@@ -1,32 +1,44 @@
 import useGetState from 'api/getState';
 import Badge from 'components/Badge';
+import { NewFeaturePopover } from 'components/NewFeaturePopover/NewFeaturePopover';
+import { NewFeaturePopoverContent } from 'components/NewFeaturePopover/NewFeaturePopoverContent';
 import { FormattedTime } from 'components/Time';
-import { useAtomValue } from 'jotai';
+import { loadingMapAtom } from 'features/map/mapAtoms';
+import { useAtom, useAtomValue } from 'jotai';
 import { useTranslation } from 'react-i18next';
 import { selectedDatetimeIndexAtom, timeAverageAtom } from 'utils/state/atoms';
 
 export default function TimeHeader() {
   const { t, i18n } = useTranslation();
   const timeAverage = useAtomValue(timeAverageAtom);
   const selectedDatetime = useAtomValue(selectedDatetimeIndexAtom);
-  const { isLoading } = useGetState();
+  const { isLoading: isLoadingData } = useGetState();
+  const [isLoadingMap] = useAtom(loadingMapAtom);
+
+  const allLoaded = !isLoadingMap && !isLoadingData;
```

Comment:
Why is this needed when it wasn't before?

If it is needed, then we could use useAtomValue to match the other atom values in the component

Reply:
At the time I originally wrote this, the popover was displaying over the loading screen. However, that no longer seems to be an issue, so I'll remove the isLoadingMap check